hugo/content/News/2024-04-29.md

```diff
@@ -0,0 +1,34 @@
+---
+title: My Overwritten Name
+weight: 2 # Order in the list
```

Comment:
We don't use weight in the news tab since they are sorted by filename (ISO date)